source/userguide/dvc.rst

```diff
@@ -0,0 +1,88 @@
+DVC for Model Tracking
+======================
+
+We use Data Version Control, or DVC, in Mindmeld in order to track trained models in our applications. DVC is an
+open-source version control system for data science and machine learning projects that enables versioning of large
+files and directories in concert with Git without storing the data itself in Git.
+
+You can find more info about DVC in the `official documentation <https://dvc.org/doc>`_.
+
+The functionality of the DVC command within Mindmeld is shown in the table below.
+
+Available Options
+-----------------
+
++-----------------+------------------------------------------------------------------------+
+| **Option**      | **Description**                                                        |
++-----------------+------------------------------------------------------------------------+
+| --init          | Initializes DVC within a repository. Must be run before other options. |
++-----------------+------------------------------------------------------------------------+
+| --save          | Saves models using DVC. Run after model building finishes.             |
++-----------------+------------------------------------------------------------------------+
+| --checkout HASH | Checks out the repo state and models corresponding to a git hash.      |
+|                 | The --save option must have been run before committing the hash for    |
+|                 | model checkout to work.                                                |
++-----------------+------------------------------------------------------------------------+
+| --help          | Show the available options and their descriptions.                     |
++-----------------+------------------------------------------------------------------------+
+
+
+We will use the Home Assistant blueprint to demonstrate the various command options.
+
+
+Example: Home Assistant Blueprint
+---------------------------------
+
+Let's take a look at how model tracking works using the Home Assistant blueprint. Below we'll walk through each of
+the necessary steps starting from an empty directory.
+
+1. **Download the Home Assistant blueprint.**
+
+.. code-block:: console
+
+   mindmeld blueprint home_assistant
+
+2. **Start a new Git repository and add Home Assistant files to Git.**
+
+.. code-block:: console
+
+   git init
+   git add home_assistant/
+   git commit -m "Added home_assistant blueprint"
+
+DVC requires a source code management (SCM) tool such as Git to operate.
+
+3. **Initialize DVC in the directory.**
+
+.. code-block:: console
+
+   python -m home_assistant dvc --init
+
+This will create all the files necessary for DVC to operate.
```

Comment:
Probably it is useful to list out the files that you are expected to see here for the DVC init operation. Maybe we should even show the terminal output.